awpy/visualization/plot.py

```diff
@@ -512,21 +513,24 @@ def plot_frame_map_control(
     if map_name not in NAV:
         msg = "Map not found."
         raise ValueError(msg)
+    if plot_type not in get_args(MapControlPlotType):
+        msg = "dist_type can only be default or players"
+        raise ValueError(msg)
 
     player_positions = extract_teams_metadata(frame)
     map_control_dict = calc_parsed_frame_map_control_values(map_name, player_positions)
-    if plot_type == "players":
+    if plot_type == "default":
         plot_map_control_snapshot(
             map_name,
             map_control_dict,
-            player_pos=player_positions,
             given_fig_ax=given_fig_ax,
             save_filepath=save_filepath,
         )
-    else:
+    elif plot_type == "players":
         plot_map_control_snapshot(
             map_name,
             map_control_dict,
+            player_pos=player_positions,
             given_fig_ax=given_fig_ax,
             save_filepath=save_filepath,
         )
```

Comment:
I think here i would do something like

```
if plot_type == "players":
    ...
else:  # default
   ...
```

and then you can add `elif`s between them if you add more options.

Reply:
Yup this totally makes sense; I'll make this change. Thanks!